step_defs/createbusinesspartner_test.py::py::test_eliminar_un_business_partner
--------------------------------------------------------------------
Last Page: https://test-xweb.eurokaizen.com/Management/BusinessPartner
  Browser: chrome (107.0.5304.107)
   Driver: chromedriver (106.0.5249.61)
Timestamp: 1669751413  (Unix Timestamp)
     Date: Tuesday, November 29, 2022
     Time: 3:50:13 PM  (Hora est√°ndar de Venezuela, UTC-04:00)
--------------------------------------------------------------------
Traceback:   File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\python.py", line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\_callers.py", line 60, in _multicall
    return outcome.get_result()
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pluggy\_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\_pytest\python.py", line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pytest_bdd\scenario.py", line 174, in scenario_wrapper
    _execute_scenario(feature, scenario, request)
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pytest_bdd\scenario.py", line 144, in _execute_scenario
    _execute_step_function(request, scenario, step, step_func)
  File "C:\Users\kaizen\AppData\Local\Programs\Python\Python38\lib\site-packages\pytest_bdd\scenario.py", line 114, in _execute_step_function
    return_value = step_func(**kwargs)
  File "C:\Users\kaizen\OneDrive\Escritorio\AutomatizacionUIBusinessBuilder\UI_test\step_defs\createbusinesspartner_test.py", line 306, in busco_el_partner_code_y_elimino
    raise Exception("Error: busco y presiono el boton de eliminar un business partner")

Exception: Error: busco y presiono el boton de eliminar un business partner